app/javascript/app/components/vulnerability-graph/vulnerability-graph-component.jsx

```diff
@@ -0,0 +1,109 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import CircularChart from 'components/circular-chart';
+import InfoIcon from 'components/info-icon';
+import { COUNTRIES_NUMBER } from 'data/country-codes';
+import styles from './vulnerability-graph-styles.scss';
+
+class VulnerabilityGraph extends PureComponent {
+  // eslint-disable-line react/prefer-stateless-function
+
+  renderCircular() {
+    const { sectionData } = this.props;
+    const value =
+      sectionData.data && Math.floor(sectionData.data.value * 10) / 10;
+    return sectionData.data ? (
+      <div className={styles.circularChartContainer}>
+        <InfoIcon
+          infoOpen={false}
+          toggleInfo={() => console.info('open')}
+          right
+          noPadding
+        />
+        <CircularChart value={value} />
+        <div className={styles.absoluteRank}>
+          <span>{sectionData.data.rank.absolute}</span>
+          <span className={styles.countriesNumber}> of {COUNTRIES_NUMBER}</span>
+        </div>
+      </div>
+    ) : (
+      'No data'
+    );
+  }
+
+  renderLine() {
+    const { sectionData } = this.props;
+    const value =
+      sectionData.data && Math.floor(sectionData.data.value * 10) / 10;
+    const riskValue = parseInt(sectionData.data.rank.absolute, 10) * 100;
```

Comment:
When we have more than 100% the line is out of the graph, could we add a limit a 100% limit?
![image](https://user-images.githubusercontent.com/10500650/31986989-981955fa-b96a-11e7-8994-654bb176ee8f.png)

Reply:
We shouldn't have more than 100% 😮  That's pretty risky